mkContainer/mkContainer.nix

```diff
@@ -71,7 +127,7 @@ let
         "FLOX_SOURCED_FROM_SHELL_RC" = "1"; # don't source from shell rc (again)
         "_FLOX_FORCE_INTERACTIVE" = "1"; # Required when running podman without "-t"
         "FLOX_SHELL" = "${containerPkgs.bashInteractive}/bin/bash";
-        "FLOX_RUNTIME_DIR" = "/tmp/flox/run";
+        "FLOX_RUNTIME_DIR" = "/run/flox";
```

Comment:
blocking: we should add a note to flox-containerize.md and the docstrings in manifest.rs explaining that we also use this to ensure the user exists in the container

Reply:
Are you talking about the `/run/flox` directory? Because that's just there so that Flox has a place to write to. It's chown'd by the user if it's added.

Reply:
Sorry just left the comment on the last line in the diff. I mean we should change the docs for `config.user`